src/main/java/nextstep/courses/domain/Base.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.courses.domain;
+
+import java.time.LocalDateTime;
+
+public class Base {
+    
+    protected LocalDateTime createdDate;
```

Comment:
```suggestion
    private final LocalDateTime createdDate;
```
부모 객체 필드 또한 private으로 구현
값을 접근할 필요가 있으면 protected getter 메서드를 통해 접근